Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -2451,3 +2451,115 @@ M  END
     outs.flush();
   }
 }
+
+TEST_CASE("test the options that toggle isotope labels", "[drawing]") {
+  SECTION("test all permutations") {
+    auto m = "[1*]c1cc([2*])c([3*])c[14c]1"_smiles;
+    std::regex regex("^<text\\s+.*>\\d</text>$");
+    std::smatch match;
+    REQUIRE(m);
+    std::string line;
+    std::string textIsoDummyIso;
+    std::string textNoIsoDummyIso;
+    std::string textIsoNoDummyIso;
+    std::string textNoIsoNoDummyIso;
+    size_t nLinesIsoDummyIso;
+    size_t nLinesNoIsoDummyIso;
+    size_t nLinesIsoNoDummyIso;
+    size_t nLinesNoIsoNoDummyIso;
+    {
+      MolDraw2DSVG drawer(300, 300, -1, -1, true);
+      drawer.drawMolecule(*m);
+      drawer.finishDrawing();
+      textIsoDummyIso = drawer.getDrawingText();
+      std::ofstream outs("testIsoDummyIso.svg");
+      outs << textIsoDummyIso;
+      outs.flush();
+      std::istringstream ins(textIsoDummyIso);
+      for (nLinesIsoDummyIso = 0; std::getline(ins, line);) {
```

Comment:
There's nothing technically wrong with this construct, but I just don't see the benefit in abusing a for loop that way...
Isn't this simpler and equivalent?
```suggestion
      nLinesIsoDummyIso = 0;
      while( std::getline(ins,line) ){
```

Reply:
This isn't robust w.r.t. output formatting changes since I didn't change the regex, but I'm not feeling up to digging through the C++ regex docs to figure out how to do the equivalent of `findall`